hrm.py

```diff
@@ -110,46 +128,43 @@ def average(time, begin_time, end_time):
 
     if begin_time >= end_time:
         raise ValueError('Begin time is before end time')
-    if time[len(time)-1] < end_time:
+    if time[len(time) - 1] < end_time:
         raise ValueError('End time occurs outside of range of csv file')
-    if time[len(time)-1] < begin_time:
+    if time[len(time) - 1] < begin_time:
         raise ValueError('Begin time occurs outside of range of csv file')
 
     begin = 0
     end = 0
 
     for i in range(1, len(time)):
-        if (time[i - 1]/1000 == begin_time):
+        if time[i - 1] / 1000 == begin_time:
             begin = i - 1
-        elif (time[i - 1]/1000 < begin_time and time[i]/1000 > begin_time):
+        elif time[i - 1] / 1000 < begin_time and time[i] / 1000 > begin_time:
             begin = i
-        if (time[i - 1]/1000 == end_time):
+        if time[i - 1] / 1000 == end_time:
             end = i - 1
-        elif (time[i - 1]/1000 < end_time) and (time[i]/1000 > end_time):
+        elif (time[i - 1] / 1000 < end_time) and (time[i] / 1000 > end_time):
             end = i
 
     time_count = 0
 
     for k in range(begin + 1, end + 1):
-        time_count = time_count + (time[k] - time[k-1])/1000
+        time_count = time_count + (time[k] - time[k - 1]) / 1000
 
-    #print (time_count)
     div = end - begin
 
     if div == 0:
         raise ValueError('Begin and End time are too close')
 
     time_avg = time_count / div
 
-    #print (60/time_avg)
-
     return 60 / time_avg
 
 
 def anomaly(time, brady_thresh, brady_time, tachy_thresh, tachy_time):
```

Comment:
nit: generally name functions using actions instead as a noun or entity (that should be reserved for classess). Something like `find_...` may be apt here